.commitlintrc.yaml

```
@@ -0,0 +1,26 @@
+extends:
+    - '@commitlint/config-conventional'
+rules:
+    # Scope enum should be customized to match project needs
+    scope-enum:
+        - 2
+        - always
+        - []
```

Let's add "deps", so linting will not fail for Dependabot commits:

```suggestion
        - ['deps']
```